pkg/kyverno/common/common.go

```diff
@@ -1062,3 +1056,27 @@ func GetPatchedResourceFromPath(fs billy.Filesystem, path string, isGit bool, po
 
 	return patchedResource, nil
 }
+
+// Setting request.operation value equal to CREATE by default
+func Set_default_value_request_operation_to_create(matches [][]string, globalValMap map[string]string) map[string]string {
```

Comment:
This doesn't match golang naming conventions. Can we please keep this to camel case?